app/serializers/api/admin/for_order_cycle/enterprise_serializer.rb

```diff
@@ -32,7 +32,6 @@ def products_scope
     if order_cycle.prefers_product_selection_from_coordinator_inventory_only?
       products_relation = products_relation.visible_for(order_cycle.coordinator)
     end
-    products_relation.order(:name)
```

Comment:
Am attempting to get the tests done to keep this one moving - this line needs to remain AFAICT.

Reply:
Hi Steve, yes, I thought the implicit return would work but it doesnt. I added it again now. But removed the order(), it's not needed here.